pkgs/by-name/sy/systemd-lsp/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  rustPlatform,
+  versionCheckHook,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "systemd-lsp";
+  version = "0-unstable-2025-06-25";
+
+  src = fetchFromGitHub {
+    owner = "JFryy";
+    repo = "systemd-lsp";
+    rev = "9834de8b338796755e5d3bd0dede5075c11662b9";
+    sha256 = "sha256-xKVmpjcKAvoyBUd7W5sl72YiPtj/7U315IkbCz0njHw=";
+  };
+
+  cargoHash = "sha256-bYksgHTXomeEJuSk800+/PYXzMvrixSjfPnoqxStWAA=";
+
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
+
+  passthru = {
+    updateScript = nix-update-script {
+      extraArgs = [ "--version=branch" ];
+    };
+  };
+
+  meta = {
+    description = "Language server implementation for systemd unit files made in Rust";
+    homepage = "https://github.com/JFryy/systemd-lsp";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ mahyarmirrashed ];
```

```suggestion
    maintainers = with lib.maintainers; [ mahyarmirrashed ];
    mainProgram = "systemd-lsp";
```
Is that the right executable name?